packages/v4/patternfly-docs/content/design-guidelines/components/spinner/spinner.md

```diff
@@ -11,11 +11,13 @@ Spinners are used to demonstrate data loading. Spinners are offered in different
 
 <img src="./img/spinner-sizes.png" alt="Visual of different spinner sizes" width="436"/>
 
+### When to use
```

Comment:
Ideally we'd break down the when to use and when not to use sections in bullet points so it's easier to consume!